client/styles.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+    font-family: "GraphikWebfont";
+    src: url('/fonts/graphik-regular-webfont.eot');
+    src: url('/fonts/graphik-regular-webfont.eot') format('embedded-opentype'),
+         url('/fonts/graphik-regular-webfont.woff') format('woff'),
+         url('/fonts/graphik-regular-webfont.ttf')  format('truetype'),
+         url('/fonts/graphik-regular-webfont.svg') format('svg');
+    font-weight: normal;
+    font-style: normal;
+  }
+
+li {
+	display: inline-block;
+    float: right;
+    padding-left: 18px;
+}
+
+#seller-wrapper {
+    width: 980px;
+    height: 131px;
+    background: #f5f5f1;
+    display: block;
+    margin-top: 28px;
+    padding: 0 28px;
+    color: #222;
+    font-family: "Graphik Webfont",-apple-system,BlinkMacSystemFont,"Roboto","Droid Sans","Segoe UI","Helvetica",Arial,sans-serif;
```

Comment:
I think you need to change it to

+    font-family: "GraphikWebfont",-apple-system,BlinkMacSystemFont,"Roboto","Droid Sans","Segoe UI","Helvetica",Arial,sans-serif;

since you wrote 
+    font-family: "GraphikWebfont"; 

in the font-face.  just no space when you try to call the font in font-family.